DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/BasicConnectorBinding.cs

```diff
@@ -0,0 +1,67 @@
+using System.Reflection;
+using Sentry.Reflection;
+using Speckle.Connectors.ArcGIS.Utils;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Models;
+using Speckle.Connectors.DUI.Models.Card;
+
+namespace Speckle.Connectors.ArcGIS.Bindings;
+
+public class BasicConnectorBinding : IBasicConnectorBinding
+{
+  public string Name { get; set; } = "baseBinding";
+  public IBridge Parent { get; set; }
+
+  public BasicConnectorBindingCommands Commands { get; }
+
+  private readonly ArcGISDocumentStore _store;
+
+  public BasicConnectorBinding(ArcGISDocumentStore store, IBridge parent)
+  {
+    _store = store;
+    Parent = parent;
+    Commands = new BasicConnectorBindingCommands(parent);
+
+    _store.DocumentChanged += (_, _) =>
+    {
+      Commands.NotifyDocumentChanged();
+    };
+  }
+
+  public string GetSourceApplicationName()
+  {
+    return "ArcGIS";
+  }
+
+  public string GetSourceApplicationVersion() => "3";
```

Comment:
We might pass these info from `ArcgisSettings` class by making one. See Alan's Rhino implementation. And it is same for also `GetSourceApplicationName`

Reply:
Done! (by Kat, I can't take credit)